resources/assets/uad_lists.json

```
@@ -37723,5 +37723,285 @@
     "neededBy": [],
     "labels": [],
     "removal": "Expert"
+  },
+  "com.android.internal.systemui.navbar.hidegestural": {
+    "list": "Aosp",
+    "description": "Allows 'Gesture hint' to be disabled in Navigation bar > Swipe gestures.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  "com.android.internal.systemui.navbar.transparent": {
+    "list": "Aosp",
+    "description": "Allows 'Transparent navigation bar' to be enabled in Developer options.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  "com.android.phone.overlay.motcommon": {
+    "list": "Oem",
+    "description": "Related to Motorola's custom overlay for Phone and other UI elements.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.android.systemui.accessibility.accessibilitymenu.auto_generated_rro_product__": {
+    "list": "Aosp",
+    "description": "Product RRO for Accessibility menu.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.android.wifi.resources.overlay.motCommon": {
+    "list": "Oem",
+    "description": "Related to Motorola's custom overlay for Wi-Fi connections.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.android.wifi.resources.overlay.WifiResScanCountryCode": {
+    "list": "Aosp",
+    "description": "Related to overlay for Wi-Fi scanning for country code.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.google.android.health.connect.backuprestore": {
+    "list": "Google",
+    "description": "Health Connect for Android backup/restore.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  "com.google.android.healthconnect.controller": {
+    "list": "Google",
+    "description": "Health Connect by Android gives you a simple way to share data between your health, fitness, and wellbeing apps without compromising on privacy.\nOnce you've downloaded Health Connect, you can access it through your settings by going to Settings > Apps > Health Connect, or from your Quick Settings menu.\nhttps://play.google.com/store/apps/details?id=com.google.android.apps.healthdata"
```

```suggestion
    "description": "Health Connect by Android gives you a simple way to share data between your health, fitness, and wellbeing apps without compromising on privacy.\nOnce you've downloaded Health Connect, you can access it through your settings by going to Settings > Apps > Health Connect, or from your Quick Settings menu.\nhttps://play.google.com/store/apps/details?id=com.google.android.apps.healthdata",
```

You forgot about this.